ez_openai/__init__.py

```diff
@@ -127,13 +130,16 @@ def ask(
         message: str | None,
         image_url: str | None = None,
         image_file: bytes | None = None,
+        additional_instructions: Optional[str] | NotGiven = NOT_GIVEN,
```

Comment:
How about this?:

```suggestion
        additional_instructions: Optional[str] = None,
```

Reply:
The OpenAI lib distinguishes between None and NotGiven and wasn't sure why.
```
additional_instructions: Optional[str] | NotGiven = NOT_GIVEN,
```

Reply:
https://github.com/openai/openai-python/blob/6316b7c0f182877e578a3dbed2fd129b63538af5/src/openai/resources/beta/threads/runs/runs.py#L1653